datarobot_batch_scoring/batch_scoring.py

```diff
@@ -772,7 +807,8 @@ def run_batch_predictions(base_url, base_headers, user, pwd,
                           out_file, keep_cols, delimiter,
                           dataset, pred_name,
                           timeout, ui, fast_mode, auto_sample,
-                          dry_run, encoding, skip_dialect):
+                          dry_run, encoding, skip_dialect,
+                          max_batch_size=3.5 * 1024 ** 2):
```

Comment:
It can be replaced with module level global patched during tests, but i'm not sure it has a sense. It's just like syntetic time passed as parameter to functions to avoid using real-time clock to allow testability...